xml/docker_dockerizing_application.xml

```diff
@@ -314,81 +314,161 @@ ENTRYPOINT ["apache2ctl"]</screen>
+       to see the available modules and extensions.</para>
+     </step>
+     <step>
+      <para>Make sure you have installed the &sle; images from &suse; Registry
+       as described in <xref linkend="Building_Pre-build_Images"/>.</para>
+     </step>
+     <step>
+      <para>Save the <filename>Dockerfile</filename> from <xref
+       linkend="ex.docker.dockerfile.apache2"/> into the
+       <filename class="directory">docker</filename> directory.</para>
+     </step>
+     <step>
+      <para>Copy the registered repositories and service files to the
+      <filename class="directory">docker/etc</filename> directory:</para>
+      <screen>cd docker
+mkdir etc
+sudo cp -a /etc/zypp/{repos.d,services.d} etc/</screen>
+      <para>This step is needed to expose the registered repositories
+       (<systemitem>SLE-Module-Server-Applications15-Pool</systemitem>) and
+       services on your host for your &docker; container.
+       If wanted, it is possible to copy only the needed repository or service
+       files to the &docker; container.
+      </para>
+     </step>
+     <step>
+      <para>Create your HTML files inside <filename>docker/data</filename>.
+       Anything you put in this directory is copied to the &docker; image
+       and as such exposed on your Web server.
+      </para>
+     </step>
+    </substeps>
+   </step>
+   <step xml:id="st.docker.apache2.build">
+    <para>Build the container. Set a tag for your image with the <option>-t</option>
+     option (here <literal>&exampleuser_plain;/apache2</literal>, but
+     you can use any name you want):</para>
+    <screen>sudo docker build -t &exampleuser_plain;/apache2 .</screen>
+    <para>&deng; refreshes repositories and installs the Apache2
+     server as it is not installed by default in the &slsa; &docker; image.</para>
+   </step>
+   <step>
+    <para>Run the image in <quote>detached</quote> mode:</para>
+    <screen>docker run --detach --interactive --tty &exampleuser_plain;/apache2</screen>
+    <para>&deng; responds with the container ID, for example:</para>
+    <screen>7bd674eb196d330d50f8a3cfc2bc61a243a4a535390767250b11a7886134ab93</screen>
+   </step>
+   <step>
+    <para>Open a Web browser and enter in the text field <uri>http://localhost:80/test.html</uri>.
+     You should see the output <literal>The Web Server is running</literal>.
     </para>
-   </listitem>
-   <listitem>
-    <para>
-     Here you copy a configuration file to the particular directory. Make sure
-     that the <filename>main.cf</filename> is located in the same directory as
-     the <filename>Dockerfile</filename>. Bear in mind that the configuration
-     will be the same for all instance run in the future. In case you need a
-     different configuration for each container, you will need to edit it after
-     running the container.
+   </step>
+  </procedure>
+  <para>With the last procedure, you have built an image which was used
```

Comment:
The above procedure describes how to build a image containing the Apache2 Web server. You can use the resulting container to serve your data with the Apache2 Web server by following these steps: